trompace/mutations/musicrecording.py

```diff
@@ -1,12 +1,138 @@
+from trompace.exceptions import UnsupportedLanguageException, NotAMimeTypeException
+from trompace.mutations.templates import format_mutation, format_link_mutation
+from trompace import StringConstant, _Neo4jDate, check_required_args, filter_none_args, docstring_interpolate
+from trompace.constants import SUPPORTED_LANGUAGES
 
 
-def mutation_create_musicrecording():
-    pass
+MUSICRECORDING_ARGS_DOCS = """name: The name of the MusicRecording object.
+        creator: The person, organization or service who created the thing the web resource is about.
+        contributor: A person, an organization, or a service responsible for contributing the MusicRecording to the web resource. This can be either a name or a base URL.
+        format_: A MimeType of the format of the page describing the MusicRecording.
+        source: The URL of the web resource about this MusicRecording.
+        title: The title of the resource indicated by `source`
+        """
 
 
-def mutation_update_musicrecording():
-    pass
+@docstring_interpolate("musicrecording_args", MUSICRECORDING_ARGS_DOCS)
+def mutation_create_musicrecording(*, title: str, contributor: str, creator: str, source: str, format_: str, encodingformat: str = None,
+                                   name: str = None, language: str = None, description: str, date: str = None, subject: str = None):
+    """Returns a mutation for creating a MusicRecording object.
+    https://schema.org/MusicRecording
+
+    Arguments:
+        {musicrecording_args}
+
+    Returns:
+        The string for the mutation for creating the MusicRecording.
+    """
+
+    if "/" not in format_:
+        raise NotAMimeTypeException(format_)
+
+    if language is not None and language not in SUPPORTED_LANGUAGES:
+        raise UnsupportedLanguageException(language)
```

Comment:
we need a call to `check_required_args` here. This checks that someone doesn't explicitly pass in None for a value, e.g. `mutation_create_musicrecording(title=None)`